preprocessing/data_transformer.py

```diff
@@ -213,3 +212,80 @@ def encode_categorical_features(dataframe_dict: dict, columns_list: list,
         pd.set_option('display.max_rows', None)
 
     return dataframe_dict
+
+
+def filter_category(value: str,
+                    to_remove: set) -> str:
+    """ Category filter
+
+    This function resets the value to NAN it it's in corresponding set
+
+    :param str value: a value to be checked.
+    :param set to_remove: a set of values to be reset.
+
+    :return:
+            value: an original value or the NAN-value.
+    """
+    if value in to_remove:
```

Comment:
Usually, we need one line empty line as a space between the end of the docstring and the first code. This is important for the documentation later.